template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": {
+    "text": "Web Application (Razor Pages)",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
```

Comment:
This is the package ID for the template engine package, which is where we are putting Visual Studio localizable resources for the release. We just need to use a unique id here. Think of this as the name/descriptions in a .vstemplate file, but with having a fallback to use the "text" value if the resource look up fails.

For this template, go ahead and use 1017 and 1018 for the name and description strings, respectively, and I'll add those ids to the resource file in VS with the values that are in this PR.